device/tests/Microsoft.Azure.Devices.Client.Test/DeviceClientTests.cs

```diff
@@ -1147,5 +1147,15 @@ public void DeviceClient_OnConnectionClosed_RecoverThrowsException_InvokeHandler
             Assert.AreEqual(ConnectionStatus.Disabled, status);
             Assert.AreEqual(ConnectionStatusChangeReason.Client_Close, statusChangeReason);
         }
+
+        [TestMethod]
+        [TestCategory("DeviceClient")]
+        public void DeviceClient_ProductInfo_stores_product_info()
```

Comment:
Nit:
- Naming should be of form: <Unit>_<Action>_<Expected result>. E.g. `ProductInfo_StoresProductInfo_Ok`.
- Each word should be capitalized.

Reply:
Will do. I find this test naming scheme bewildering...is there a doc somewhere that explains it?